hypothesis/via
# Enable the stats stub page

This turns on the stats stub page in NGINX.

To test locally:

 * `docker stop via3_nginx-proxy_1; docker rm via3_nginx-proxy_1; make services;`
 * Go to: http://localhost:9083/_stats
 * You should see something like:

```
Active connections: 1 
server accepts handled requests
 2 2 7 
Reading: 0 Writing: 1 Waiting: 0
```
